src/utils/load-pdf.tsx

```diff
@@ -0,0 +1,15 @@
+/** @jsx createElementEntity */
+import * as pdfjs from "pdfjs-dist";
+import { HubsWorld } from "../app";
+import { createElementEntity, renderAsEntity } from "../utils/jsx-entity";
+
+export function* loadPDF(world: HubsWorld, url: string) {
+  const pdf = (yield pdfjs.getDocument(url).promise) as pdfjs.PDFDocumentProxy;
+
+  // TODO Should we create the canvas texture here and load/render the first page?
```

Comment:
In my opinion, we should not. Instead, we should just let the pdf system load whatever page needs to be loaded. If for some reason a page takes a long time to load, oh well. If a page throws an error while loading, we'll have to handle that somehow. (Currently, we don't.) Either way, it seems nice to keep the system simple by having only one place that pages are loaded rather than doing another page load here. Thoughts?